modules/core/tests/SettingsControllerTest.php

```
@@ -10,4 +10,99 @@
 #[CoversClass(SettingsController::class)]
 class SettingsControllerTest extends TestCase
 {
+    #[Test]
+    public function it_get_settings_index_requires_authentication(): void
+    {
+        $this->markTestIncomplete('HTTP test infrastructure needed');
+    }
```

PHPMD (2.15.0)

[error] The method it_get_settings_index_requires_authentication is not named in camelCase. (undefined)

(CamelCaseMethodName)